src/components/appSwitch/AppSwitch.js

```diff
@@ -241,24 +238,31 @@ const AppSwitch = (props: LoadingProps) => {
 
   useEffect(() => {
```

Comment:
this hook is unnecessary - as the hooks with deps also will be called at first render

Reply:
I'll test, I think it's necessary when the user is not logged in and after he signs in is pushed to a path.

Reply:
It's necessary.

Reply:
so please a comment in code explaining the issue, so in the future other devs know not to remove it